src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.Android.targets

```
@@ -35,4 +35,8 @@
 		<_UnoProjectSystemPackageReference Include="Uno.WinUI.Graphics2DSK" ProjectSystem="true" />
 	</ItemGroup>
 
+	<ItemGroup Condition="$(_IsExecutable) == 'true' AND $(UnoFeatures.Contains(';skiarenderer;')) AND $(UnoFeatures.Contains(';mauiembedding;'))">
+		<_UnoProjectSystemPackageReference Include="Uno.Extensions.Maui.WinUI.Runtime.Skia" PrivateAssets="all" IncludeAssets="None" ProjectSystem="true" />
```

IncludeAssets is not needed here, none would have the effect of not referencing the assembly from the package

```suggestion
		<_UnoProjectSystemPackageReference Include="Uno.Extensions.Maui.WinUI.Runtime.Skia" PrivateAssets="all" ProjectSystem="true" />
```